.circleci/config.yml

```diff
@@ -30,6 +30,9 @@ jobs:
     executor: go
     steps:
       - checkout
+      - run:
+          name: Install packer
+          command: go get -u github.com/gobuffalo/packr/packr
```

Comment:
we should fix an exact version not the master one

Reply:
`go get` is not very handy for pulling fixed versions... you need to do some trickery with git in the $GOPATH :( . Not sure best option for us... maybe we can build a small docker image from `cirleci/golang` installing golangci-lint and packer 🤔